website/docs/plugins/optuna_sweeper.md

```diff
@@ -0,0 +1,46 @@
+---
+id: optuna_sweeper
+title: Optuna Sweeper plugin
+sidebar_label: Optuna Sweeper plugin
+---
+
+[![Example application](https://img.shields.io/badge/-Example%20application-informational)](https://github.com/facebookresearch/hydra/tree/master/plugins/hydra_optuna_sweeper/example)
```

Comment:
could you add the plugin doc to the plugin sidebar? 
https://github.com/facebookresearch/hydra/blob/master/website/sidebars.js#L85-L93


To view your doc locally, you can follow the developer guide here
https://hydra.cc/docs/next/development/documentation

Reply:
I added "Optuna Sweeper Plugin" to the sidebar in https://github.com/facebookresearch/hydra/pull/1132/commits/1dbb9c79fabc83e30daf99d604774424aee295d7. I think it is rendered correctly in my environment. Please let me know if something is wrong with it in your environment.

![image](https://user-images.githubusercontent.com/3255979/98921958-39407a80-2515-11eb-9000-7747e4ff7351.png)

Reply:
Looks good. Remember to complete the top links once the plugin is released. (link to pypi package, link to downloads etc).